README.md

```diff
@@ -80,6 +80,50 @@ public class MyClass {
 
 Protobuf (via `com.google.protobuf:protobuf-java`) objects and schema is supported out of the box.
 
+### Protobuf example
+
+```protobuf
+syntax = "proto3";
+package org.projectnessie.cel.toolstests;
+
+message MyPojo {
+  string Property1 = 1;
+}
+```
+
+```java
+public class MyClass {
+  public Boolean evalWIthProtobuf() {
+    ScriptHost scriptHost = ScriptHost.newBuilder().build();
+
+    Script script =
+        scriptHost
+            .buildScript("inp.Property1 == checkName")
+            .withDeclarations(
+                // protobuf types need the type's full name
+                Decls.newVar("inp", Decls.newObjectType(Dummy.MyPojo.getDescriptor().getFullName())),
```

Comment:
might be unclear where `Dummy` is coming from, doesnt it depend on the proto filename?
in our tests its clear in the example it isnt imo

Reply:
Updated the readme (this "Dummy" came from the .proto file name in the test)